From 345c5fbce56bb4191a2a23f15ee3827384ab5727 Mon Sep 17 00:00:00 2001
From: Surajit Podder <spodder@codeaurora.org>
Date: Thu, 20 Nov 2014 18:16:33 +0530
Subject: [PATCH 49/95] NuPlayer: Disable widevine flag for L3 content

Disable widevine flag for L3 content. This is
needed because Generic source considers widevine
playback to be L1 by default, without checking if
secure buffers are requested by the widevine source.

Change-Id: I4e33836dfa98554d5ae0cb8cc01c1c960ba86fb1
---
 media/libmediaplayerservice/nuplayer/GenericSource.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/media/libmediaplayerservice/nuplayer/GenericSource.cpp b/media/libmediaplayerservice/nuplayer/GenericSource.cpp
index e4cd338..ddaee78 100644
--- a/media/libmediaplayerservice/nuplayer/GenericSource.cpp
+++ b/media/libmediaplayerservice/nuplayer/GenericSource.cpp
@@ -240,6 +240,8 @@ status_t NuPlayer::GenericSource::initFromDataSource() {
                     if (mUIDValid) {
                         extractor->setUID(mUID);
                     }
+                } else {
+                     mIsWidevine = false;
                 }
             }
         }
-- 
2.5.0

